setup.py

```diff
@@ -74,11 +74,7 @@
     author_email="admin@apeworx.io",
     url="https://github.com/ApeWorX/ape-infura",
     include_package_data=True,
-    install_requires=[
-        "eth-ape>=0.8.1,<0.9",
-        "web3>=7.6.0,<8",
-        "requests>=2.32.3,<3",
-    ],
+    install_requires=["eth-ape>=0.8.1,<0.9", "web3>=6.20.3,<8", "requests>=2.28.1,<3"],
```

Comment:
nit: if you end the last entry with a comma, it will keep each item on a newline, which is a bit easier for auditing